Templates/MinimalProject/Template/project.json

```
@@ -12,5 +12,7 @@
     ],
     "icon_path": "preview.png",
     "engine": "o3de",
-    "external_subdirectories": []
+    "external_subdirectories": [
+        "Code"
+    ]
```

Shouldn't this directory be "Gem" now?
```suggestion
    "external_subdirectories": [
        "Gem"
    ]
```